3rdparty/libbacktrace/dwarf.c

```diff
@@ -1854,10 +1854,12 @@ read_line_program (struct backtrace_state *state, struct dwarf_data *ddata,
 		  }
 	      }
 	      break;
+#ifdef DW_LNE_set_discriminator
```

Comment:
upstream this please

Reply:
Upstream libbacktrace has its own definitions of all the constants (as enums) since the initial commit there. Sounds like the libbacktrace copy here is either from before git, or has been modified to use the system dwarf headers.

Reply:
ah - I need to update my copy then. thanks for the heads-up!